api/dscinitialization/removedServices/servicemesh_types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package removedservices
```

Comment:
Use the "minimal" API approach we have used in other PRs to make sure the corresponding CRDs are not generated.